src/chrome/Link.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import classNames from 'classnames'
+import { Link } from 'react-router-dom'
+
+const defaultColorClassName = 'text-qriblue hover:text-qriblue-800'
```

Comment:
no need for a `const` here, declare `'text-qriblue hover:text-qriblue-800'` directly as a default on the `Link` signature